src/conspiracies/corpusprocessing/umap_hdb.py

```diff
@@ -69,6 +69,7 @@ def filter_triplets_with_stopwords(
 
 def load_triplets(
     file_path: str,
+    language: str = "danish",
```

Comment:
only keep one notion of languages? ("da")

Reply:
Agree. As mentioned in another comment, the `umap_hdb` comes from Stine's branch, and it's up for a refactoring to streamline stuff like this.